cheatsheets/OS_Command_Injection_Defense_Cheat_Sheet.md

```diff
@@ -40,7 +62,11 @@ If there are available libraries or APIs for the language you use, this is the p
 
 **TODO: To enhance.**
```

Comment:
Can / should we remote this "TODO" now that more content is present? I'm not sure how much more was expected here.

Reply:
@h4r1337 can you look into that?

Reply:
Well, I'm not so sure how much more was expected there. I removed `escapeshellcmd()` becasue generally `escapeshellarg()` is considered more secure than `escapeshellcmd()` because it is designed to escape individual command line arguments rather than entire command lines.
I also described how the function behaves.
if the idea that we can utilise language-specific functions to escape values supplied to OS commands is what this section is supposed to convey, then it is done there with a PHP function as an example.

Reply:
This looks solid to me, solid enough. I'm pushing it live, please stop me or ask me to reverse it @kwwall and others if I am in error.